src/utils/time.js

```diff
@@ -1,11 +1,15 @@
 const DAY = 86400000
 
-export const extractTimeFromDate = (date) =>
-  date.toLocaleString('en-GB', {
+export const extractTimeFromDate = (datetime) => {
+  if (typeof datetime === 'string') {
+    datetime = new Date(datetime)
+  }
```

Comment:
just a suggestion.
maybe we can add a test to check` if` statement, pass string and check that it returns exactly what we want ?